src/cdi_CPEloss/Analytic_Spitzer_Eloss_Model.hh

```diff
@@ -0,0 +1,48 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   cdi_CPEloss/Analytic_Spitzer_Eloss_Model.hh
+ * \author Ben R. Ryan
+ * \date   Feb 21 2020
+ * \brief  Analytic_KP_Alpha_Eloss_Model class definition.
```

Comment:
Looks like a copy-paste leftover :)

Reply:
That'll teach me to scrub wikipedia for stopping power source code...